build.rs

```diff
@@ -49,7 +49,7 @@ fn main() -> Result<()> {
     let target = env::var("TARGET").expect("TARGET was not set");
     let mut base_config = cc::Build::new();
 
-    language_std(&mut base_config, "c++17");
+    language_std(&mut base_config, "c++20");
```

Comment:
Try the suggested solution from the compiler:
```suggestion
    language_std(&mut base_config, "c++2a");
```